cli/crates/backend/assets/grafbase.default.config.ts

```diff
@@ -0,0 +1,54 @@
+import { g, auth, config } from '@grafbase/sdk'
+
+// Welcome to Grafbase!
+// Define your data models, integrate auth, permission rules, custom resolvers, search, and more with Grafbase.
```

Comment:
Should we use an Oxford comma here?

Reply:
It is copied from `schema.graphql`, we should be using them there too.